net/openssh/Portfile

```diff
@@ -162,6 +162,11 @@ if {${name} eq ${subport}} {
         }
     }
 
+    pre-test {
+        ui_msg "Tests require a cooperating server named 'somehost'."
+        ui_msg "Failure to connect to it results in a hang."
+    }
+
```

Comment:
Why does it require a remote host? One could just connect to an unprivileged server spun up and listening on an unusual port on localhost, no?

Reply:
@pmetzger 
> Why does it require a remote host? One could just connect to an unprivileged server spun up and listening on an unusual port on localhost, no?

Don't shoot the messenger. :-)

This message is simply reporting the upstream brain damage, in hopes of saving others the trouble of letting `port test` sit there for a long time, thinking that it's a really time-consuming test, when in reality it's just a failure to meet the unspecified prerequisites, in combination with a "hang on failure" test design.

And yes, the server really does need to be named 'somehost', presumably by adding an entry to `/etc/hosts`.  It's not even a well-localized reference:
```
MacPro:openssh-portable fw$ git grep -w somehost | wc -l
     237
```

Most references are to just 'somehost', but there are also a few of these:
```
MacPro:openssh-portable fw$ git grep -w somehost.example.com | wc -l
       3
```